src/generic.jl

```diff
@@ -166,6 +170,30 @@ inverse(t)(y) == inverse(t, y) == inverse(transform(t))(y)
 """
 inverse(t::AbstractTransform) = Base.Fix1(inverse, t)
 
+"""
+$(SIGNATURES)
+
+Return the log prior correction used in [`transform_and_logjac`](@ref). The second
+argument is the output of a transformation.
+
+The log jacobian determinant is corrected by this value, usually for the purpose of
+making a distribution proper. Can only be nonzero when [`nonzero_logprior`](@ref) is
+true.
+"""
+logprior(t::AbstractTransform, y) = 0.0
```

Comment:
The literal `Float64` seems unfortunate as it could cause undesired promotions. Maybe change to `false` or use a similar approach as for `logjac_zero`?